### PR TITLE
B 18161 int update email customer submitted moves

### PR DESCRIPTION
**Pull request by taeJungCaci**
## Summary

Updating email template that is sent when the customer creates a move with any shipment type.

- Fixed a typo from "an" to a"
- Added a sentence "If you have requested a PPM, DO NOT start your PPM until your counselor has approved it in MilMove. You will receive an email when that is complete."
- Changed email signature to "USTRANSCOM MilMove Team"

### How to test

For Gov't Counseling email:
1. Create a new customer with your email and add any shipment to the move.
2. Once you have finished submitting, you should receive an email with a title: "Thank you for submitting your move details.

For GHC Counseling email:
1. Create a new customer with your email, Marine Corps, and NS Mayport, FL 32228 as the duty location (Might have to pick 32233 as zip for your address if it doesn't work).
2. Choose any shipment for the move and finish submitting.
3. Once you have finished submitting, you should receive an email with a title: "Thank you for submitting your move details.

## Screenshots after the update

Gov't Counseling:
image

GHC Counseling:
image

### CODE REVIEW

**Comment by deandreJones** [comment]
> Note: I think we are allowed to approve and merge in without server_test_coverage passing. At least from what I can see from other merged-in PRs.

that is correct- server_Test_coverage does NOT have to pass, caveat, that there is sufficient testing for any new or changed code

File: pkg/assets/notifications/templates/move_submitted_template.html [hunk 1]
```diff
@@ -81,8 +85,8 @@ <h4>
   Thank you,
 </p>
```
Comment (WeatherfordAaron) [inline pkg/assets/notifications/templates/move_submitted_template.html:86]: I recommend adding a <br> tag between the "Thank you," and the "USTRANSCOM MilMove Team" in stead of the separate <p> sections due to the fact that it places a larger space between the <p> sections than a simple linebreak.

**Review by WeatherfordAaron — APPROVED** [review]
lgtm

File: pkg/assets/notifications/templates/move_submitted_template.html (hunk 1 above)
Comment (taeJungCaci) [reply pkg/assets/notifications/templates/move_submitted_template.html]: Thanks for your feedback Aaron, I have updated as suggested!

**Review by brooklyn-welsh — APPROVED** [review]
LGTM 👍

**Pull request merged**